apps/web/src/containers/TradingContest/points-dashboard/points-dashboard.tsx

```diff
@@ -0,0 +1,105 @@
+import { Box, ThemeProvider, styled } from '@mui/material';
+import { THEME_VARIANTS } from '@notional-finance/util';
+import {
+  SectionTitle,
+  OuterContainer,
+  BgImgContainer,
+  MainContainer,
+} from '../components';
+import { colors, useNotionalTheme } from '@notional-finance/styles';
+import backgroundColors from '../assets/color-blobs.png';
+import { YourPointsOverview } from './your-points-overview';
+import { PointsDashboardManager } from './points-dashboard-manager';
+import { FormattedMessage } from 'react-intl';
+
+export const PointsDashboard = () => {
+  const theme = useNotionalTheme(THEME_VARIANTS.DARK, 'landing');
+  return (
+    <ThemeProvider theme={theme}>
+      <OuterContainer sx={{ paddingBottom: '0px' }}>
+        <BgImgContainer>
+          <img src={backgroundColors} alt="bg img" />
+        </BgImgContainer>
+        <OpacityBG>
+          <MainContainer>
+            <TopContentContainer>
+              <DisplayBoxWrapper>
+                <SectionTitle sx={{ marginBottom: theme.spacing(3) }}>
+                  <FormattedMessage defaultMessage={'Season 2'} />
+                </SectionTitle>
+                <DisplayBox>April 15 - July 15</DisplayBox>
```

Comment:
Can you put the seasons in a constants file and then look up the appropriate date here?